.github/workflows/podvm.yaml

```diff
@@ -61,3 +61,17 @@ jobs:
           podvm/${{ matrix.dockerfile }}
         build-args: |
           "CLOUD_PROVIDER=${{ matrix.provider }}"
+
+    - name: Extract Asset
+      run: |
+        UUID=$(cat /proc/sys/kernel/random/uuid | cut -c 1-8)
+        CONTAINER_NAME=temp-container-${UUID}
+        CID=$(docker create --name $CONTAINER_NAME quay.io/confidential-containers/podvm-${{ matrix.provider }}-${{ matrix.os }}:${{ github.sha }} /bin/sh)
+        podvm=$(docker export $CID | tar t | grep podvm)
+        docker cp $CONTAINER_NAME:/$podvm podvm-${{ matrix.provider }}-${{ matrix.os }}-${{ github.sha }}.qcow2
```

Comment:
nit: `podman save ${IMG_ID} | tar -xO --no-wildcards-match-slash '*.tar' | tar -x` worked for me but i'm fine with that as well